pushkin/database/database.py

```diff
@@ -226,6 +224,7 @@ def delete_login(login):
         session.delete(device)
     session.delete(login)
     session.commit()
+    session.close()
```

Comment:
It would be nice if there is a way to write some annotation or something that automatically opens a session and closes/commits at the end of the method to minimize chance for leaking the sessions. Maybe open this as an improvement